front/src/modules/activities/notes/components/NoteCard.tsx

```diff
@@ -71,17 +76,24 @@ export const NoteCard = ({
     ?.content.map((x: any) => x.text)
     .join('\n');
 
+  const fieldContext = useMemo(
+    () => ({ recoilScopeId: note?.id ?? '' }),
+    [note?.id],
+  );
+
   return (
-    <StyledCard>
-      <StyledCardDetailsContainer
-        onClick={() => openActivityRightDrawer(note.id)}
-      >
-        <StyledNoteTitle>{note.title ?? 'Task Title'}</StyledNoteTitle>
-        <StyledCardContent>{body}</StyledCardContent>
-      </StyledCardDetailsContainer>
-      <StyledFooter>
-        <ActivityRelationEditableField activity={note} />
-      </StyledFooter>
-    </StyledCard>
+    <FieldContext.Provider value={fieldContext as GenericFieldContextType}>
```

Comment:
There's currently a bug on main, where if you try to edit the relations of a note from the notes list on the notes tab, all the relations of all the cards will open. This is because in order to open the relation editor, the code utilises the hook `useInlineCell`. `useInlineCell` utilizes the `FieldContext` to extract the `recoilScopeId` which it then uses to determine which of the editors to open. This also affected creation of new notes by simply adding a relation. This change mainly fixes that.